src/shell.rs

```diff
@@ -94,6 +98,22 @@ mod tests {
     }
 
     #[apply(opts)]
+    #[cfg(windows)]
+    fn cmd_cmd(cmd: Option<&str>, hook: InitHook, echo: bool, resolve_symlinks: bool) {
```

Comment:
This won't run as `#[cfg(feature = "nix-dev")]` is in effect. Should I pull it to a separate module?